diff --git a/node_modules/react-native-text-recognition/android/build.gradle b/node_modules/react-native-text-recognition/android/build.gradle
index b76e895..6da162d 100644
--- a/node_modules/react-native-text-recognition/android/build.gradle
+++ b/node_modules/react-native-text-recognition/android/build.gradle
@@ -18,11 +18,11 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('TextRecognition_compileSdkVersion', 29)
-    buildToolsVersion safeExtGet('TextRecognition_buildToolsVersion', '29.0.2')
+    compileSdkVersion safeExtGet('TextRecognition_compileSdkVersion', 36)
+    buildToolsVersion safeExtGet('TextRecognition_buildToolsVersion', '36.0.0')
     defaultConfig {
-        minSdkVersion safeExtGet('TextRecognition_minSdkVersion', 16)
-        targetSdkVersion safeExtGet('TextRecognition_targetSdkVersion', 29)
+        minSdkVersion safeExtGet('TextRecognition_minSdkVersion', 24)
+        targetSdkVersion safeExtGet('TextRecognition_targetSdkVersion', 36)
         versionCode 1
         versionName "1.0"
 
@@ -56,6 +56,6 @@ repositories {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation "com.facebook.react:react-native:+"  // From node_modules
-    implementation 'com.google.android.gms:play-services-mlkit-text-recognition:16.3.0'
+    implementation 'com.google.android.gms:play-services-mlkit-text-recognition:19.0.1'
 
 }
diff --git a/node_modules/react-native-text-recognition/android/src/main/java/com/reactnativetextrecognition/TextRecognitionModule.java b/node_modules/react-native-text-recognition/android/src/main/java/com/reactnativetextrecognition/TextRecognitionModule.java
index 64b41a0..8cb1787 100644
--- a/node_modules/react-native-text-recognition/android/src/main/java/com/reactnativetextrecognition/TextRecognitionModule.java
+++ b/node_modules/react-native-text-recognition/android/src/main/java/com/reactnativetextrecognition/TextRecognitionModule.java
@@ -26,7 +26,7 @@ import com.google.mlkit.vision.common.InputImage;
 import com.google.mlkit.vision.text.Text;
 import com.google.mlkit.vision.text.TextRecognition;
 import com.google.mlkit.vision.text.TextRecognizer;
-import com.google.mlkit.vision.text.TextRecognizerOptions;
+import com.google.mlkit.vision.text.latin.TextRecognizerOptions;
 
 @ReactModule(name = TextRecognitionModule.NAME)
 public class TextRecognitionModule extends ReactContextBaseJavaModule {
